packages/mdc-textfield/label/_mixins.scss

```diff
@@ -46,14 +48,30 @@
   }
 }
 
-@include mdc-text-field-invalid-label-shake_keyframes_(standard, 100%);
-@include mdc-text-field-invalid-label-shake-keyframes_(box, 50%);
-@include mdc-text-field-invalid-label-shake-keyframes_(box-dense, 90%, 0%, .923);
-@include mdc-text-field-invalid-label-shake_keyframes_(outlined, 130%);
-@include mdc-text-field-invalid-label-shake_keyframes_(outlined-dense, 145%, 0%, .923);
-@include mdc-text-field-invalid-label-shake_keyframes_(outlined-leading-icon, 130%, 27%);
-@include mdc-text-field-invalid-label-shake_keyframes_(outlined-leading-icon-dense, 145%, 25%, .923);
-@include mdc-text-field-invalid-label-shake-keyframes_(textarea, 50%, 0%, .923);
+@include mdc-text-field-invalid-label-shake_keyframes_(standard, $mdc-text-field-label-position-y);
+@include mdc-text-field-invalid-label-shake-keyframes_(box, $mdc-text-field-box-label-position-y);
+@include mdc-text-field-invalid-label-shake-keyframes_(box-dense, $mdc-text-field-box-dense-label-position-y, 0%, $mdc-text-field-dense-label-scale);
+@include mdc-text-field-invalid-label-shake_keyframes_(outlined, $mdc-text-field-outlined-label-position-y);
+@include mdc-text-field-invalid-label-shake_keyframes_(outlined-dense, $mdc-text-field-outlined-dense-label-position-y, 0%, $mdc-text-field-dense-label-scale);
+@include mdc-text-field-invalid-label-shake_keyframes_(outlined-leading-icon, $mdc-text-field-outlined-label-position-y, $mdc-text-field-outlined-with-leading-icon-label-position-x);
+@include mdc-text-field-invalid-label-shake_keyframes_(outlined-leading-icon-dense, $mdc-text-field-outlined-dense-label-position-y, $mdc-text-field-outlined-dense-with-leading-icon-label-position-x, $mdc-text-field-dense-label-scale);
+@include mdc-text-field-invalid-label-shake-keyframes_(textarea, $mdc-text-field-textarea-label-position-y, 0%, $mdc-text-field-textarea-label-scale);
+
+@mixin mdc-text-field-label-float_($positionY, $positionX: 0%, $scale: .75) {
```

Comment:
Should the default of $positionX also be 0, since we are checking 0? I ran npm run build and tested it, and It looks like it works as intended. Really just a style decision.

Reply:
I copied the arguments from `mdc-text-field-invalid-label-shake-keyframes_`...no idea if its the "right" decision, but at least we're consistent